v-next/hardhat/src/internal/cli/helpers/utils.ts

```diff
@@ -0,0 +1,124 @@
+import type { ParameterType } from "@nomicfoundation/hardhat-core/config";
+import type { Task } from "@nomicfoundation/hardhat-core/types/tasks";
+
+export const GLOBAL_OPTIONS = [
```

Comment:
Just a note: I think we can handle this and reserved words differently so that they are more unified with the rest of the global options.

For a follow up pr though

Reply:
I created an issue for this